### PR TITLE
Remove redundant properties

### PR DESCRIPTION
**Pull request by Spappz**
Not too many files, so please double-check that I'm understanding correctly!

### CODE REVIEW

File: animations/spells/1st/chilling-spray.json [hunk 1]
```diff
@@ -5,8 +5,6 @@
 			"preset": "template",
 			"file": "jb2a.cone_of_cold.blue",
 			"contents": [
-				{
```
Comment (MrVauxs) [inline animations/spells/1st/chilling-spray.json:8]: This adds another animation "as is". The intent here is to fill the entire 90° template with the animation which is only 30/45°

File: animations/spells/1st/breathe-fire.json [hunk 2]
```diff
@@ -5,8 +5,6 @@
 			"preset": "template",
 			"file": "jb2a.burning_hands.01.orange",
 			"contents": [
-				{
```
Comment (MrVauxs) [inline animations/spells/1st/breathe-fire.json:8]: As chilling spray

File: animations/spells/1st/ignition.json [hunk 3]
```diff
@@ -9,7 +9,6 @@
 					"file": "jb2a.cast_generic.fire.side01.orange.0",
 					"options": {
 						"waitUntilFinished": -1200,
-						"rotateTowards": {},
```
Comment (MrVauxs) [inline animations/spells/1st/ignition.json:12]: I will have to check but this should most likely be substituted for "true" basically.

File: animations/spells/1st/ignition.json [hunk 4]
```diff
@@ -44,7 +42,6 @@
...
```
Comment (MrVauxs) [inline animations/spells/1st/ignition.json:47]: As earlier comment.

File: animations/spells/1st/kinetic-ram.json [hunk 5]
```diff
@@ -10,7 +10,6 @@
 					"value": 3,
 					"considerTokenScale": true
 				},
-				"rotateTowards": {},
```
Comment (MrVauxs) [inline animations/spells/1st/kinetic-ram.json:13]: As earlier comment.

**Review by MrVauxs — CHANGES_REQUESTED** [review]

File: animations/spells/1st/chilling-spray.json (hunk 1 above)
Comment (Spappz) [reply animations/spells/1st/chilling-spray.json:8]: Got it. Removing the nonempty-object check on the base schema and only applying it to the top-level object then.

**Pull request merged**